src/components/Common/Tooltip/index.tsx

```diff
@@ -1,16 +1,51 @@
 import clsx from "clsx";
-import React from "react";
+import React, { useEffect } from "react";
 
 const Tooltip: React.FC<{
   children?: React.ReactNode;
   tooltip: React.ReactNode;
   className?: string;
-}> = ({ children, tooltip, className }) => {
+  wrapperClassName?: string;
+}> = ({ children, tooltip, className, wrapperClassName }) => {
+  const wrapperRef = React.useRef<HTMLSpanElement>(null);
+  const tooltipRef = React.useRef<HTMLSpanElement>(null);
+
+  useEffect(() => {
+    function fixPosition() {
```

Comment:
would this function be better defined outside the scope of `useEffect`, i.e. defined using `useCallback` ?